internal/model/client_test.go

```diff
@@ -7,34 +7,67 @@ import (
 )
 
 func TestClient_GetStatus(t *testing.T) {
+	status := func(s string) RefData { return RefData{Label: s} }
+
 	tests := []struct {
-		orderStatuses []string
-		wantStatus    string
+		orders     []Order
+		orderType  string
+		wantStatus string
 	}{
 		{
-			orderStatuses: []string{"Closed", "Open", "Duplicate", "Active", "Closed", "Open", "Duplicate"},
-			wantStatus:    "Active",
+			orders: []Order{
+				{Status: status("Closed"), Type: "pfa"},
+				{Status: status("Open"), Type: "pfa"},
```

Comment:
I don't know if it matters for this test but I've set the ordertype so it'll only be set on active orders. I.e. if there's a closed/ duplicate PFA it won't be 'pfa' order type it'll be null. Let me know if this is an issue or you want to chat about it.

Reply:
hmm but how do we know that the order is/was a pfa after it gets closed?

Reply:
Sorry I didn't phrase that super clearly, I've set it on the client. So it will tell you what the casetype is for them (i.e. what order they have that's active, including Dual/ Hybrid). We might need to have a longer chat if you're expecting this info to be on the order as well? I was assuming `type` here is the active case type I've saved on the client?

Reply:
OK that makes sense. `Type: "pfa"` in this context is the caseSubtype of the order, which should always have a value. The Caseload view needs to display the order status of the most recent HW order. It's completely separate from activeCaseType 🙂

Reply:
Ah apologies I got confused